src/symposium-homepage.html

```diff
@@ -130,9 +130,7 @@
 
       <p> On the 1st of May, our symposium about Cyber Security will take place. This symposium, organized by the symposium committee of W.I.S.V. 'Christiaan Huygens' will take place in Theater de Veste, Delft.</p>
 
-      <p>During the day, there will be two separated tracks, one specialized on the mathematical aspects and one specialized on computer science side.</p>
-
-      <p>At the end of the day, we will finish with a discussion about the ethics behind cyber security, followed by drinks and the possibility to network and share thoughts on the discussed topics. </p>
+      <p>During the day, there will be two separated tracks and at the end of the day, we will finish with a discussion about the ethics behind cyber security, followed by drinks and the possibility to network and share thoughts on the discussed topics. </p>
```

Comment:
I would change 'two separated tracks' to 'two separate tracks'. There is also a trailing space at the end of the sentence.